lib/cmps/base.ts

```diff
@@ -140,7 +140,9 @@ export class AutoConsent extends AutoConsentBase {
 
   async _runRulesSequentially(tab: TabActor, rules: AutoConsentRuleStep[]): Promise<boolean> {
     for (const rule of rules) {
+      console.log('Running rule...', rule, tab.id);
```

Comment:
You've added quite a bit of logging with `console.log` throughout this PR. I'd prefer if we trimmed this down a bit, and where we do have logging put it behind a flag, so it is off by default.

Reply:
Agreed. These messages are really helpful during debugging, but I noticed they were quite noisy in jenkins runs. For the user it shouldn't matter because these messages are printed in the background page